docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,114 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the [SQL `Merge` statement](/sql/merge). ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection of tokens when OAuth 2.0 refresh-tokens are
```

Comment:
Sorry ... you lost me.. that sentence makes no sense to me. Maybe just

Fix potential incorrect rejection OAuth 2.0 refresh tokens.